docs/layouts/shortcodes/generated/checkpointing_configuration.html

```diff
@@ -18,7 +18,7 @@
             <td><h5>state.backend.local-recovery</h5></td>
             <td style="word-wrap: break-word;">false</td>
             <td>Boolean</td>
-            <td>This option configures local recovery for this state backend. By default, local recovery is deactivated. Local recovery currently only covers keyed state backends. Currently, the MemoryStateBackend does not support local recovery and ignores this option.</td>
+            <td>This option configures local recovery for this state backend. By default, local recovery is deactivated. Local recovery currently only covers keyed state backends. Currently, the OperatorStateBackend does not support local recovery and ignores this option.</td>
```

Comment:
I think these two lines mean the same:
```
Local recovery currently only covers keyed state backends. 
```
```
Currently, the OperatorStateBackend does not support local recovery and ignores this option.
```
We DO have operator state backend in the code, but it's not exposed to the user (it's not possible to set state.backend to operator for example). So I'd just remove the 2nd sentence.

Reply:
Apart from removing the 2nd sentence, I think we can also emphasize like this:

~~~
Local recovery currently only covers keyed state backends (including both EmbeddedRocksDBStateBackend and HashMapStateBackend).
~~~

Reply:
Currently, ChangelogStateBackend also supports local recovery. 
If we want to highlight them, maybe also adds ChangelogStateBackend ?

Reply:
I'm not sure about ChangelogStateBackend because it is also not exposed to the user as a backend (but rather as a runtime feature).